server/service/osquery_utils/queries.go

```diff
@@ -305,11 +309,18 @@ FROM logical_drives WHERE file_system = 'NTFS' LIMIT 1;`,
 	"google_chrome_profiles": {
 		Query:            `SELECT email FROM google_chrome_profiles WHERE NOT ephemeral`,
 		DirectIngestFunc: directIngestChromeProfiles,
-		// Technically this does work on Windows and Linux, but so far no one is
-		// deploying the extension to those platforms and it's causing log spam
-		// for customers. See https://github.com/fleetdm/fleet/issues/4123
-		Platforms: []string{"darwin"},
+		Discovery:        discoveryTable("google_chrome_profiles"),
 	},
+	"orbit_info": {
+		Query:            `SELECT * FROM orbit_info`,
+		DirectIngestFunc: directIngestOrbitInfo,
+		Discovery:        discoveryTable("orbit_info"),
+	},
+}
+
+// discoveryTable returns a query to determine whether a table exists or not.
+func discoveryTable(tableName string) string {
+	return fmt.Sprintf("SELECT 1 FROM osquery_registry WHERE active = true AND registry = 'table' AND name = '%s';", tableName)
```

Comment:
Ah, nice. This query won't generate error logs when the table doesn't exist.

In the future I wonder if we'll want to extend this function to support multiple table names.

Reply:
> In the future I wonder if we'll want to extend this function to support multiple table names.

+1, it's possible.